src/cascade/saver/create_model_version.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from cascade.core import getLoggers, __version__
+from cascade.core.db import cursor
+
+CODELOG, MATHLOG = getLoggers(__name__)
```

Comment:
Do we make it in the saver or at the start of the model? It might be at the start because the rest of the machinery assumes an mvid and uses that to decide where to save files in a separate directory.

Reply:
I was imagining it at the end but you're totally right, the tier 3 promotion code needs an mvid and that's at the beginning.

Reply:
That's fine. We can put this code wherever when it's clearer how to make the `main()` that they can use. It's great to have it.

Reply:
Yeah, can't have this in the main version. It's something they have to do deliberately.